lib/src/bloc/movies_bloc.dart

```diff
@@ -0,0 +1,29 @@
+import 'dart:async';
+
+import '../model/movies_model.dart';
+import '../resources/movies_repository.dart';
+import 'IMoviesFetcherBloc.dart';
```

Comment:
keep the same convention for the name of the files, lower case separated with low hyphens, for example: i_movies_fetcher_bloc.

But in this case the interface should have the same name as the class, less the "i", you should rename it to IMoviesBloc and the file i_movies_bloc, to keep consistency between files

Reply:
Changed